pkg/api/api.proto

```diff
@@ -199,9 +201,3 @@ message Application {
   repeated Release releases = 2;
   string team = 3;
 }
-
-message Commit {
```

Comment:
:thinking: In my mental model, it really makes sense to have these information always together ( when was something changed and by whom ) and so having this type seems like a good idea. On the other hand, the author for a release will always be a bot user but in that case we could just keep the author name and email empty. Do you think it makes sense to keep this type around under a different name?

Reply:
I tried to keep it at first and have a different name for this structure. I decided to remove it because to me it felt more simpler to have the information there without this extra layer. I would agree with you if it was used in more places. but it only makes sense as the `lock_commit` or in the context of locks. 

I actually hadn't thought of keeping the name and email empty. I can keep this under a different name, e.g. `Metadata`?

i am only against writing the name and email of release author (which is as you said, bots) in the manifest repo. (as part of the scripts)